### PR TITLE
Add configuration to enable Allstar.

### PR DESCRIPTION
**Pull request by jeffmendoza**
@bcoe Please take a look at the list of repos.

### CODE REVIEW

**Review by bcoe — APPROVED** [review]
This looks great to me, I'm excited to try it out.

If I'm understanding, we'll start seeing a failing check on PRs if the repository is out of compliance?

Is the Allstar app already installed on the org?

**Review by JustinBeckwith — APPROVED** [review]
Excited to see this in action!

**Comment by jeffmendoza** [comment]
>If I'm understanding, we'll start seeing a failing check on PRs if the repository is out of compliance?

Currently it just creates issues. Blocking PRs is a planned "action" for when we want to get more serious about compliance.

> Is the Allstar app already installed on the org?

Yes, OSPO went ahead and installed in on all the orgs, but it isn't enabled to do any actions until you add these files.

File: allstar/allstar.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  disableRepoOverride: true
+  optInRepos:
+  - nodejs-asset
+  - nodejs-automl
+  - nodejs-bigtable
+  - nodejs-dialogflow
+  - nodejs-firestore
+  - nodejs-logging
+  - nodejs-notebooks
+  - nodejs-storage
```
Comment (bcoe) [inline allstar/allstar.yaml]:
```suggestion
  - nodejs-asset
  - nodejs-bigtable
  - nodejs-notebooks
```

**Review by bcoe — APPROVED** [review]
Thanks for making this PR, after talking with Justin just trimmed down the starting repositories, to keep this as a very limited test.

**Pull request merged**